### ISSUE TITLE
Python Code Completion not working

### ISSUE THREAD

**Issue by f3rn4nd0000** [VSCodium/vscodium#940]
**Describe the bug**

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
In any python code try to get the suggestions by pressing Ctrl+Space, none shows

**Expected behavior**
It should be able to show me some code suggestions (auto-complete)

**Screenshots**

![vscodiumbug](https://user-images.githubusercontent.com/42982873/145710332-595421a8-15ce-48f4-b296-c86d41dc64bd.gif)

My product.json
```
{
  "extensionsGallery": {
    "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
    "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
    "itemUrl": "https://marketplace.visualstudio.com/items",
    "controlUrl": "",
    "recommendationsUrl": ""
  },
  "extensionAllowedProposedApi": [
    "ms-python.python"
    // ...
  ], 
}
```
Previously i did setup Jedi instead of PyLance as language server, as suggested by @clayrisser here: https://github.com/VSCodium/vscodium/issues/892
and my settings.json had these lines:
```
{
    "enableJedi": true,
    "python.languageServer": "Jedi",
    "python.formatting.provider": "autopep8"
}
```
besides the default ones obviously, but i don't remember if the update broke it or not, not sure tbh, but it stopped working so i removed these lines trying to make it work after i modified settings.

**Desktop (please complete the following information):**
 - OS: [Pop!_OS 21.04]
 - Architecture [x.64]
 - Version [1.63.0]

**Additional context**
Happy to help, just tell me what i need to do.

**Comment by AminGholizad** [comment]
I have the same problem and unfortunately I had to switch back to vs code 😞

**Comment by bxff** [comment]
Yeah I had the same issue, took me 4 hours but I found a good solution.

The problem is that Pylance is closed source code so VSCodium doesn't allow it. Also Pylance licence may not prohibit it to run inside VSCodium, though I cannot say for sure: https://marketplace.visualstudio.com/items/ms-python.vscode-pylance/license

@asdorsey solution works partially but it doesn't work inside Jupyter, and also misses a lot of functionality of Pylance I'd avoid if I can.

Though if you want to still install Pylance you can do so this way, the only downside is it doesn't update on its own, and you'd need to update it from time to time:
* Install Python extensions like Python, Jupyter as you do in general in VSCodium,
* Install Pylance by downloading the extensions `*.vsix`([Download Extention](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)),
* Change the setting `Python: Language Server` from default to `Pylance`
<img width="1016" alt="Screenshot 2022-12-08 at 8 33 25 AM" src="https://user-images.githubusercontent.com/51504045/206345878-2a533dc6-50d4-493c-9491-8b54469a2788.png">
<img width="402" alt="Screenshot 2022-12-08 at 8 37 36 AM" src="https://user-images.githubusercontent.com/51504045/206346403-028fd33e-d304-43ee-a601-9f73b3ba76a4.png">

@AminGholizad

**Comment by RoyiAvital** [comment]
@bxff , Will `PyLance` be updated automatically once it is installed that way?

**Comment by GitMensch** [comment]
> Will `PyLance` be updated automatically once it is installed that way?

No, vscode doesn't update anything that was installed from vsix. Pylance isn't open source so it isn't on open-vsx - the only option would be to switch the marketplace, too, via product json. ... but then: if you want to use the MS marketplace and MS proprietary extensions you may be better with their Visual Studio Code binaries (and disable telemetry via settings) - if they provide binaries for your OS.

**Comment by daiyam** [comment]
To manage VSIX from different marketplaces, you can use https://open-vsx.org/extension/zokugun/vsix-manager
But it will be up to you to test if the VSIX are compatible with your editor.

**Comment by GitMensch** [comment]
Can we close this as unplanned - because we just can't do anything about this as it is an external extension 1 that works out of the box but doesn't work as extension 2 that wants itself only be used in Visual Studio Code (o "of course there is a different result").
